pkg/reconciler/events/event.go

```diff
@@ -102,22 +102,3 @@ func EmitError(c record.EventRecorder, err error, object runtime.Object) {
 		c.Event(object, corev1.EventTypeWarning, EventReasonError, err.Error())
 	}
 }
-
-// EmitEvent emits an event for object if afterCondition is different from beforeCondition
```

Comment:
These were added because of backward compatibility,  but since we're cleaning up all unnecessarily exported methods, I'd say it makes sense to remove them.